### ISSUE TITLE
Form field color contrast

### ISSUE THREAD

**Issue by lukefretwell** [proudcity/wp-proudcity#1901]
> Placeholder text for 2 form fields fails color contrast requirements: "How can we help you?" and "enter an address or location". Both test at 2.84:1 and the minimum requirement is 4.5:1. 
> Recommend removing placeholder text altogether because it causes additional accessibility issues and adding normal form label above each field.

![Screen Shot 2021-10-25 at 12 32 29 PM](https://user-images.githubusercontent.com/1569463/138758510-8a1fa24d-15ef-4c9d-9e9d-500a25fb4df9.png)

## Comments

Agree.
